### PR TITLE
Add notices about configuring Meilisync primary key

### LINKED ISSUE #2598: Meilisync guides: Add note regarding primary key field
A user mentioned they ran into issues with using `meilisync` when their dataset contains several fields ending in `_id`:

> […] I had to add the pk  (primary key) attribute to the tables in the config.yml because my tables have more than one column ending in 'id'. Note that the pk  YAML attribute is not mentioned in the documentation, but the reported error suggests its existence.

Pinging @meilisearch/devrel-team, the official maintainers of the `meilisync` guides.

## Tasks
- [ ] verify with SME (@alallema) whether the info checks out and if this scenario also applies to PostgreSQL
- [ ] update `/learn/primary_db_sync/meilisync_mysql.mdx`
- [ ] if necessary, update `/learn/primary_db_sync/meilisync_postgre.mdx`

## References
- User report on Discord: https://discord.com/channels/1006923006964154428/1156967511313621022/1166073598579048559
- SME: @alallema

### PR DESCRIPTION
**Pull request by Strift**
# Pull Request

## Related issue
Fixes #2598 

## What does this PR do?
- Add a notice about the field to configure the primary key for meilisync

### CODE REVIEW

File: learn/primary_db_sync/meilisync_mysql.mdx [hunk 1]
```diff
@@ -152,6 +153,10 @@ sync:
 
 Replace `MYSQL_TABLE_NAME_1` and `MEILISEARCH_INDEX_NAME_1` with the names of your tables and indexes. As the above example demonstrates, you may sync multiple tables.
 
+**Primary key inference**
```
Comment (guimachiavelli) [inline learn/primary_db_sync/meilisync_mysql.mdx:156]:
```suggestion
### Primary key inference
```

File: learn/primary_db_sync/meilisync_postgresql.mdx [hunk 2]
```diff
@@ -160,6 +161,10 @@ sync:
 
 Replace `POSTGRES_TABLE_NAME_1` and `MEILISEARCH_INDEX_NAME_1` with the names of your tables and indexes. As the above example demonstrates, you may sync multiple tables.
 
+**Primary key inference**
```
Comment (guimachiavelli) [inline learn/primary_db_sync/meilisync_postgresql.mdx:164]:
```suggestion
### Primary key inference
```

**Review by guimachiavelli — CHANGES_REQUESTED** [review]
Just a minor change so we're using proper heading elements. Only bold might become an accessibility issue since it is harder for screen readers to interpret a paragraph is a new subsection when we only use bold to mark up the heading.

Alternatively, you could try to create another `<Capsule>`, but not sure we want to have two of those so close to each other.

**Comment by Strift** [comment]
Thanks for the recommendation. Commited the suggested changes!

**Comment by Strift** [comment]
I'm not aware of any reason to hold this so I would say "yes" :)

**Pull request merged**